src/st/class-wpml-pb-reuse-translations.php

```diff
@@ -138,6 +138,16 @@ private function is_same_location_and_different_ids( array $current_string, arra
 		return $current_string['location'] == $leftover_string['location'] && $current_string['id'] != $leftover_string['id'];
 	}
 
+	/**
+	 * @param string $old_text
+	 * @param string $new_text
+	 *
+	 * @return bool
+	 */
+	private function is_similar_text( $old_text, $new_text ) {
+		return 50 < WPML_ST_Diff::get_sameness_percent( $old_text, $new_text );
```

Comment:
@strategio Please don't use yodo here. It makes it hard to read and is not needed. You only need yodo conditions when comparing with variables.

Reply:
Done @brucepearson... you mean yoda :)